src/test/java/com/wikia/webdriver/common/core/SkinHelper.java

```diff
@@ -0,0 +1,31 @@
+package com.wikia.webdriver.common.core;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SkinHelper extends BasePageObject{
+
+  @FindBy(css = "body.skin-oasis")
+  private WebElement oasisClassInBody;
+  @FindBy(css = "body.ember-application")
+  private WebElement mercuryClassInBody;
+
+  public SkinHelper(WebDriver driver) {
+    super(driver);
+  }
+
+  public boolean isSkin(Skin skin) {
```

Comment:
This method can not be called without creating the object, so it can not be static